### ISSUE TITLE
Broken typing in latest (0.3.2)

### ISSUE THREAD

**Issue by pejter** [aviramha/ormsgpack#24]
I saw you updated maturin in #23 so it can generate type stubs for you but there was no release build triggered and as such the latest version on PyPI still has broken types, because it's missing the "py.typed" file.

I don't know if you'd prefer to include the "py.typed" file or remove the `.pyi` stub and let maturin generate everything, but either way it'd be nice to release a fixed version. If you let me know which you'd prefer I can make a quick PR.

**Comment by aviramha** [comment]
Hi!
Sorry for this. I didn't think it was worth a release.
I prefer the latter - if you'll open a PR I'll create a release after merging it.
Thanks 🙏

**Comment by pejter** [comment]
So it appears maturin can't generate it's own `.pyi` from Rust's types, my bad. But it does pick up the existing definition as-is.
However there is a problem I encountered when it comes to Python version flags in PyO3. At least in my environment using the existing `#[cfg(not(python37))]` doesn't work, however `#[cfg(not(Py_3_7))]` does, as per the [PyO3 docs](https://pyo3.rs/v0.14.1/building_and_distribution/multiple_python_versions.html#conditional-compilation-for-different-python-versions).
I've created a PR for it, feel free to merge or reject based on your own findings.
